bin/fasta-to-phylip.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import sys
+import tempfile
+
+from dark.fasta import FastaReads
+from dark.fasta_ss import SSFastaReads
+from dark.fastq import FastqReads
+
+
+if __name__ == '__main__':
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        description=('Given FASTA on stdin write the sequence ids and lengths '
+                     'to stdout.'))
+
+    parser.add_argument(
+        '--readClass', default='fasta', choices=('fasta', 'fastq', 'fasta-ss'),
```

Comment:
This code is copied from filter-fasta.py :-)   I understand why it's confusing, as it literally is the class of reads in the file, but we could have an argument named more like your suggestion. We're using this name in about 6 scripts now, always in the same way I think.  If you want to change it, let's make a ticket to do it in all scripts (and anything that calls them, like slurm pipeline scripts etc).